seatunnel-e2e/seatunnel-connector-v2-e2e/connector-kafka-e2e/src/test/java/org/apache/seatunnel/e2e/connector/kafka/KafkaIT.java

```diff
@@ -752,6 +753,50 @@ public void testKafkaProtobufToAssert(TestContainer container)
                 });
     }
 
+    @TestTemplate
+    public void testKafkaToKafkaExactlyOnce(TestContainer container)
```

Comment:
Suggest add STREMING task

Reply:
Thank you for your suggestion, I will update the code shortly.